docs/getting-started.md

```diff
@@ -0,0 +1,54 @@
+# Getting started
+
+Now that you know what KLoadGen is and that you are familiar with its components, you can install the plugin and put it to work.
+
+## Prerequisites
+
+You will need these tools to start using the plugin:
```

Comment:
Would it be right to link to these tools' getting started/installation sites?

Reply:
Jose agreed on leaving this information out, as the target user for this documentation needs to have deep knowledge of these tools before using the plugin and also because there are many ways of downloading them.